dashboard/src/pages/MyChapters.vue

```diff
@@ -55,6 +58,7 @@ const scheduled_events = createListResource({
   doctype: 'FOSS Chapter Event',
   fields: ['name', 'event_name', 'event_type', 'chapter_name', 'status', 'event_start_date'],
   filters: {
+    event_start_date: ['>', new Date()],
```

Comment:
For multi day events, this will result in card vanishing on suppose day 2. 

Better logic would be to have `event_end_date > new Date()`

Reply:
I repurposed what was there in the chapter page listing, will change it